modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/71_context_api.yml

```diff
@@ -1,11 +1,17 @@
 "Action to list contexts":
+    - skip:
+        features: fips_140
+        reason: "The tests expect to be run with OSS distribution"
```

Comment:
The problem for these tests is that these expect to run in a cluster with the OSS distribution and the expected number and order of contexts, and classes order and number for the score context are different between oss and default distribution test clusters. In FIPS 140 mode we use the default distribution ( because i.e. one can't set `xpack.security.fips_mode.enabled:` in OSS ) , so these tests would fail.